modules/javafx.graphics/src/test/java/test/javafx/css/Node_cssStyleMap_Test.java

```diff
@@ -101,6 +100,7 @@ private void checkFoundStyle(Property<?> property, Map<StyleableProperty<?>, Lis
 
     }
 
+    @Ignore("JDK-8234241")
```

Comment:
Based on the description in JBS, I presume that [JDK-8234241](https://bugs.openjdk.java.net/browse/JDK-8234241) is a test bug? If so, can you label the new bug as such?

Reply:
Hi Kevin, I have updated the JBS.